pkgs/development/python-modules/langgraph/default.nix

```
@@ -21,7 +26,7 @@
 
 buildPythonPackage rec {
   pname = "langgraph";
-  version = "0.1.9";
+  version = "0.2.4";
   pyproject = true;
 
   disabled = pythonOlder "3.10";
```

```suggestion
  disabled = pythonOlder "3.9";
```
https://github.com/langchain-ai/langgraph/blob/a16a33806c60e80be1fb64b54f2e5835f6080c01/libs/langgraph/pyproject.toml#L11

---

Fixed.